src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useCallback, useEffect, useState } from "react";
+import "./Login.scss";
+import { Button, Form, Nav } from "react-bootstrap";
+import { Link, useNavigate } from "react-router-dom";
+import { useForm } from "react-hook-form";
+import { contactpattern, emailpattern } from "../../constants";
+import { useDispatch, useSelector } from "react-redux";
+import { login } from "../../redux/store/Authentication/actions";
+import { LoginStatus } from "../../models/LoginStatus";
+
+function Login() {
+  const [tabState, SetTabState] = useState("with-email");
+  const [userMainField, setuserMainField] = useState("Email address");
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const isAuthenticated = useSelector(
+    (state: any) => state.authentiaction.authenticated
+  );
+  const [displayMsg, setDisplayMsg] = useState(false);
```

Comment:
displayMsg is saving either True and false. in that case please rethink the variable name.

Reply:
Changed the variable with appropriate name